templates/index2.html

```diff
@@ -0,0 +1,442 @@
+        </div>
+        <div class="col-4">
+          <h2>Easy to update</h2>
+          <p class="p-heading--five">Snaps update automatically and transactionally so your app is always fresh.</p>
+          <p class="p-heading--five"><a href="https://docs.snapcraft.io/core/updates">Transactional updates</a></p>
+        </div>
+        <div class="col-4">
+          <h2>Safe to run</h2>
+          <p class="p-heading--five">Snaps stay secure as they are confined from the OS, other apps, and hardware functions.</p>
+          <p class="p-heading--five"><a href="https://docs.snapcraft.io/reference/confinement">Keeping your system safe</a></p>
+        </div>
+      </div>
+    </div>
+
+    <div class="p-strip is-deep is-bordered">
+      <div class="row">
+        <div class="col-6">
+          <h2>The Snap Store</h2>
+        </div>
+        <div class="col-6">
+          <p class="p-heading--five">The Snap Store is enabled by default on tens of millions Linux systems. It allows developers to release free or paid apps for multiple architectures, on multiple release channels from daily builds to stable releases.</p>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-12">
+          <h3>Featured snaps</h3>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-3">
+          <div class="p-heading-icon">
+            <div class="p-heading-icon__header">
+              <a class="p-heading-icon__img" href="/atom">
+                <img src="https://assets.ubuntu.com/v1/694b2e80-atom.png" alt="" />
+              </a>
+              <h5 class="u-no-margin"><a href="/atom">Atom</a></h5>
+            </div>
+          </div>
+        </div>
+        <div class="col-3">
+          <div class="p-heading-icon">
+            <div class="p-heading-icon__header">
+              <a class="p-heading-icon__img" href="/brackets">
+                <img src="https://assets.ubuntu.com/v1/7a350f60-brackets.png" alt="" />
+              </a>
+              <h5 class="u-no-margin"><a href="/brackets">Brackets</a></h5>
+            </div>
+          </div>
+        </div>
+        <div class="col-3">
+          <div class="p-heading-icon">
+            <div class="p-heading-icon__header">
+              <a class="p-heading-icon__img" href="/discord">
+                <img src="https://assets.ubuntu.com/v1/cbb032ba-discord.png" alt="" />
+              </a>
+              <h5 class="u-no-margin"><a href="/discord">Discord</a></h5>
+            </div>
+          </div>
+        </div>
+        <div class="col-3">
+          <div class="p-heading-icon">
+            <div class="p-heading-icon__header">
+              <a class="p-heading-icon__img" href="/go">
+                <img src="https://assets.ubuntu.com/v1/b3e93ee1-golang.png" alt="" />
+              </a>
+              <h5 class="u-no-margin"><a href="/go">Golang</a></h5>
```

Comment:
In my previous review I said “As with the language logos above, each snap icon here should be part of the same link as the text.”

This is an improvement, in that the icons are links. Unfortunately they’re separate links, with no text of their own, which is an accessibility problem: a screenreader will resort to saying something like “Graphic link: B 3 E 9 3 E E 1 dash golang dot P N G”. This could be avoided by making the icon part of the same link as the text, as you’ve done with the language logos.

Reply:
This vanilla pattern doesn't really allow that by default. Will try to work around that later.

I haven't heard that screen reader would try to read a URL of an image, I would rather expect it to read URL where link leads to ("/go" in this case). But I guess adding alt="Golang" to the image could fix that.
But that still would mean two similar links would be read by screenreader.